crates/python/qcs_sdk/_executable.pyi

```diff
@@ -22,24 +22,54 @@ class Executable:
         compile_with_quilc: Optional[bool] = None,
         compiler_options: Optional[CompilerOpts] = None,
     ) -> "Executable": ...
-    async def execute_on_qvm(self) -> ExecutionData:
+    def execute_on_qvm(self) -> ExecutionData:
         """
         Execute on a QVM which must be available at the configured URL (default http://localhost:5000).
 
         Raises:
             - ``QcsExecutionError``: If the job fails to execute.
         """
         ...
-    async def execute_on_qpu(self, quantum_processor_id: str) -> ExecutionData:
+    async def execute_on_qvm_async(self) -> ExecutionData:
         """
+        Async version of ``execute_on_qvm``.
```

Comment:
Is this supposed to be markdown or some other format? If markdown, it should be a single tick on each side (same in other places)

Reply:
I am guessing that the double backtick is used to link to other defined names (classes, functions, etc.), but I am not familiar with the syntax.

Reply:
Same answer as https://github.com/rigetti/qcs-sdk-rust/pull/252#discussion_r1113477496 - double-backtick for a cross-reference is a standard I've seen so far, though my tooling (VSCode) doesn't do anything with it. I'll make the changes to bring inconsistencies in line with the pattern, I see no clear answer.